f2/src/components/formik/button/theme.js

```diff
@@ -0,0 +1,47 @@
+export const buttonTypes = {
```

Comment:
The buttonTypes definition in theme.js may be better co-located with button/index.js. Since we are using CSS-in-JS, this style definition just like our styled, are similar in nature. All CSS applied to same components. No other components will share this definition, right?
Otherwise, our theme.js will have lots of definitions like this, unless we want to share this definition with other components.